Document NodeIterator FILTER_REJECT vs FILTER_SKIP (#34605)
mdn/content: files/en-us/web/api/document/createnodeiterator/index.md

    | `NodeFilter.SHOW_TEXT`                                   | `4`                                                     | Shows {{ domxref("Text") }} nodes.                                                                                                                                                                                                                                                                                                                                                              |

- `filter` {{optional_inline}}

  - : A callback function or an object with an `acceptNode()` method. The function or method will be called for each node in the subtree based at root which is accepted as included by the whatToShow flag to determine whether or not to include it in the list of iterable nodes. The method should return one of `NodeFilter.FILTER_ACCEPT`, `NodeFilter.FILTER_REJECT`, or `NodeFilter.FILTER_SKIP`. See the [Example](#examples).

>     For `createNodeIterator`, it is equivalent to return `NodeFilter.FILTER_REJECT`, `NodeFilter.FILTER_SKIP`, or any unrecognized value. This node will not be included in the list of iterable nodes, but its children will continue to be iterated over.

Reviewer: For me at least it is easier reading to say:

```suggestion
    For `createNodeIterator`, the values `NodeFilter.FILTER_REJECT` and `NodeFilter.FILTER_SKIP` are equivalent. This node will not be included in the list of iterable nodes, but its children will continue to be iterated over.
```

I think it's reasonable to expect the context of "for createNodeIterator" to be implicit, and  I'm not sure it helps to include "any unrecognized value".

Author: I'm fine to drop the "any value" part. Although I think it helps to be exhaustive, from my research Chrome and Firefox have diverging behavior on returning unrecognized values in `createTreeWalker` filter function, so I didn't update that page as I planned to. However I think "for `createNodeIterator` is helpful to remind readers it's not true for `createTreeWalker`.

Reviewer: > However I think "for createNodeIterator is helpful to remind readers it's not true for createTreeWalker.

Yes, I changed my mind about that :).
